### DISCUSSION TITLE
Doesn't seem to do anything ! What have I missed

### DISCUSSION THREAD

**Discussion by Rege0039** [valinet/ExplorerPatcher#2984] [Q&A]
Hi, 

Installed and appears in the taskbar ( I am using Start 11 from Stardock on Canary Windows Insider builds).  Search box appears when icon clicked on, can type in the box but no searching happens. Everything is installed and working but set not to show icon. 

What have I done wrong ?

**Comment by Amrsatrio** [comment]
Seems like you were getting an SCOOBE (Second Chance Out-of-Box Experience) and somehow EP crashed it repeatedly. Maybe try the latest prerelease, and then logging out and logging back in.

**Comment by pyrates999** [comment]
If you're running a canary build, canary builds are not supported because the code is not final and can change with each new release.
